src/components/containers/seo.tsx

```diff
@@ -64,13 +72,16 @@ const SEO = ({
     )
 
     const no_index_staging = process.env.GATSBY_ENV === 'staging'
-    const metaDescription = description || queries.site.siteMetadata.description
-    const site_url = queries.site.siteMetadata.siteUrl
+    const metaDescription =
+        (show_branding && description) || (show_branding && queries.site.siteMetadata.description)
+    const site_url = show_branding && queries.site.siteMetadata.siteUrl
     const { locale: lang, pathname } = React.useContext(LocaleContext)
     const formatted_lang = lang.replace('_', '-')
     const locale_pathname = pathname.charAt(0) === '/' ? pathname : `/${pathname}`
-    const default_og_title = localize('Online trading with Deriv | Simple. Flexible. Reliable.')
-    const default_og_description = localize('Trading platforms designed with you in mind.')
+    const default_og_title =
+        show_branding && localize('Online trading with Deriv | Simple. Flexible. Reliable.')
```

Comment:
```suggestion
        show_branding ? localize('Online trading with Deriv | Simple. Flexible. Reliable.') : ""
```
if `show_branding` is `false` then the meta tag would be `undefined`